examples/grpc-service-oauth2/grpc_service_oauth2.md

```diff
@@ -1,18 +1,47 @@
 # Service - OAuth2
 
-A gRPC service/resource can be secured with OAuth2 and by enforcing authorization optionally. Then, it validates the OAuth2 token sent in the `Authorization` metadata against the provided configurations. This calls the configured introspection endpoint to validate.
-
-Ballerina uses the concept of scopes for authorization. A resource declared in a service can be bound to one/more scope(s). The scope can be included in the introspection response using a custom claim attribute. That custom
+A gRPC service/resource can be secured with OAuth2 and by enforcing
+authorization optionally. Then, it validates the OAuth2 token sent in the
+`Authorization` metadata against the provided configurations. This calls the
```

Comment:
When we break these sentences into newlines, the website shows them in newlines even though the md file doesnt show it. So we have to keep them in the same line.

Reply:
Fixed via https://github.com/ballerina-platform/ballerina-distribution/pull/3122/commits/f59075869e675ac29a5b3aae1f1828eee43ee66f.